how-to/customize-workspace/docs/how-to-define-apps-fdc3-1-2.md

```diff
@@ -102,6 +102,8 @@ Customize workspace supports the following manifest types (for the list in code
 - **inline-window**: This manifest type expects the manifest setting to have the classic window options inline rather than a url to a json file.
```

Comment:
Looks like their is a lot of duplication with the manifest type docs, could this be a separate document